profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -113,66 +116,37 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
   if ($entity_type == 'node') {
     $node = $form_state['entity'];
     if (workbench_moderation_node_moderated($node)) {
+      // If the tmgmt_poetry module is disabled remove possibility to select
+      // multiple languages and by doing that follow the default tmgmt path.
+      if (!module_exists('tmgmt_poetry')) {
+        $form['languages']['#multiple'] = FALSE;
+      }
 
       $states = array_values(_tmgmt_workbench_available_states());
       $state = $node->workbench_moderation['current']->state;
       $translation_possible = in_array($state, $states, 'strict');
 
       $options = $form['languages']['#options'];
 
-      /** @var TMGMTJobItem[] $existing_jobs */
-      $existing_jobs = tmgmt_job_item_load_latest(
-        'workbench_moderation',
-        $entity_type,
-        $node->nid,
-        $node->translations->original
-      );
+      // Getting information about existing job items.
+      if (!isset($form_state['tmgmt_workbench']['existing_jobs'])) {
```

Comment:
I was expecting this form_state to be always clean.
But here you are counting that a value for ['tmgmt_workbench']['existing_jobs'] may exist already.

When would ['tmgmt_workbench']['existing_jobs'] have values? Just to make sure that we really want to use that, instead of using "tmgmt_job_item_load_latest".

I also see that we have two similar functions:
tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter
tmgmt_poetry_form_tmgmt_entity_ui_translate_form_alter

In the later we have the same code here... so I guess this is correct but please can you clarify?

By the way, to imagine that these two hooks will run, no specific order... OMG!